server/Controllers/changePicture.js

```diff
@@ -0,0 +1,44 @@
+import UserModel from "../Schemas/User";
+
+const createRandomId = (length) => {
+  let result = "";
+  const characters =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+  const charactersLength = characters.length;
+  for (let i = 0; i < length; i++) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
+};
```

Comment:
please, delete this horrible thing and use `crypto.crypto.randomBytes(5)` like I did in SignUp